nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -988,6 +1008,8 @@ public CombinedLibraryModels(Iterable<LibraryModels> models, Config config) {
           }
           castToNonNullMethodsBuilder.put(entry);
         }
+        nullableVariableTypeUpperBoundsBuilder.putAll(
+            libraryModels.typeVariablesWithNullableUpperBounds().entries());
```

Comment:
Can't you just do this?
```suggestion
        nullableVariableTypeUpperBoundsBuilder.putAll(
            libraryModels.typeVariablesWithNullableUpperBounds());
```